modules/launchd/default.nix

```diff
@@ -0,0 +1,179 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  inherit (pkgs.stdenv.hostPlatform) isDarwin;
+  inherit (lib.generators) toPlist;
+  inherit (hm.dag) entryAfter;
+
+  cfg = config.launchd;
+  labelPrefix = "org.nixos";
+  dstDir = "${config.home.homeDirectory}/Library/LaunchAgents";
+
+  launchdConfig = { config, name, ... }: {
+    options = {
+      enable = mkEnableOption name;
+      config = mkOption {
+        type = types.submodule (import ./launchd.nix);
+        default = { };
+        example = {
+          ProgramArguments = [ "/usr/bin/say" "Good afternoon" ];
+          StartCalendarInterval = {
+            Hour = 12;
+            Minute = 0;
+          };
+        };
```

Comment:
```suggestion
        example = literalExpression ''{
          ProgramArguments = [ "/usr/bin/say" "Good afternoon" ];
          StartCalendarInterval = {
            Hour = 12;
            Minute = 0;
          };
        }'';
```

Reply:
Thanks, I applied the changes in the latest commit.